README.md

```diff
@@ -14,7 +14,8 @@ nice visual feedback.
 
 * Visual experience, shows CMake output in a console-like window
 * Slick management of build configurations
-* Autocompletion for build targets, build configurations and tests
+* Autocompletion for build targets, build configurations and tests using the
+  `cmake-file-api(7)`
```

Comment:
Actually, the file API is only used for build targets. Also, I think this is too low-level of a detail to be mentioned in the feature overview. Perhaps we can revert this and mention the file API in the requirements section? (see comment below)

Reply:
Sure.